uaclient/entitlements/livepatch.py

```diff
@@ -221,17 +221,14 @@ def application_status(
         if not livepatch.is_livepatch_installed():
             return (ApplicationStatus.DISABLED, messages.LIVEPATCH_NOT_ENABLED)
 
-        try:
-            system.subp(
-                [livepatch.LIVEPATCH_CMD, "status"],
-                retry_sleeps=LIVEPATCH_RETRIES,
-            )
-        except exceptions.ProcessExecutionError as e:
+        if livepatch.status() is None:
             # TODO(May want to parse INACTIVE/failure assessment)
-            logging.debug("Livepatch not enabled. %s", str(e))
             return (
                 ApplicationStatus.DISABLED,
-                messages.NamedMessage(name="", msg=str(e)),
+                messages.NamedMessage(
+                    name="livepatch-client-failure",
+                    msg="canonical-livepatch status didn't finish successfully",
```

Comment:
This should in the `messages` module right ? Since we might show that message on the JSON output of  `enable` or `disable`

Reply:
You're right, and I did that but accidentally put that change in the wrong commit :facepalm: I'll fix it as I rebase to fix conflicts presently

Reply:
Updated!